src/core/Config.ts

```
@@ -568,6 +568,10 @@ export class Config {
     return !!this.getConfig('translate.deepl.enableLog')
   }
 
+  static get deeplGlossariesDir() {
+    return this.getConfig<string|null|undefined>('translate.deepl.glossariesDir');
```

ESLint

[error] Extra semicolon.

(`@typescript-eslint/semi`)

---

[error] Extra semicolon.

(semi)